CHANGELOG.md

```diff
@@ -1,11 +1,29 @@
 # Release History
-## 0.10.0 (Unreleased)
+## 0.10.0 (2022-09-16)
 
 ### New Features:
+- Added support for evaluating Snowpark dataframes asynchronously :
```

Comment:
Remove the space before the colon at the end of the sentence.

Reply:
I suggest changing `asynchronously` with `asynchronous query`. People make think `asynchronously` is the coroutine-related stuffs.

Reply:
done